### ISSUE TITLE
can cpp-package use operators in contrib directory?

### ISSUE THREAD

**Issue by xinghedyc** [apache/incubator-mxnet#8795]
1. I found the generated op.h by OpWrapperGenerator.py dosen't include contrib operators?
2. Is it possible to use c++ to write a predict code for like Deformable Conv Nets with pretrained model.

**Comment by eric-haibin-lin** [comment]
@lx75249 Does the op generator include contrib ops?

**Comment by lx75249** [comment]
No, but it's possible. I just checked the generator. Contrib ops names start with `_contrib_`, I intentionally filtered all ops starting with `_` to get rid of internal ops. 

See
https://github.com/apache/incubator-mxnet/blob/master/cpp-package/scripts/OpWrapperGenerator.py#L349
